x-pack/plugins/gis/public/shared/layers/sources/ems_tms_source/ems_tms_source.test.js

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+
+import {
+  EMSTMSSource,
+} from './ems_tms_source';
+
+describe('EMSTMSSource', () => {
+
+  it('should get attribution from markdown (tiles v2 legacy format)', async () => {
+
+    const emsTmsSource = new EMSTMSSource({
+      id: 'road_map'
+    }, {
+      id: 'road_map',
```

Comment:
`id` should not be part of dataMeta function parameter and is not needed

Reply:
yes, it is needed. it is used for matching the correct service with the service-configuration in the descriptor